cmake/Modules/FindClangFormat.cmake

```diff
@@ -12,30 +12,19 @@
 #
 # .. code-block:: cmake
 #
-# find_package(ClangFormat) 
+# find_package(ClangFormat)
 # if(ClangFormat_FOUND)
 # message("clang-format executable found: ${ClangFormat_EXECUTABLE}\n"
-#         "version: ${ClangFormat_VERSION}") 
+#         "version: ${ClangFormat_VERSION}")
 # endif()
 
 find_program(ClangFormat_EXECUTABLE
              NAMES
-                   clang-format-9
-                   clang-format-9.0
-                   clang-format-8
-                   clang-format-8.0
-                   clang-format-7
-                   clang-format-7.0
-                   clang-format-6.0
-                   clang-format-5.0
-                   clang-format-4.0
-                   clang-format-3.9
-                   clang-format-3.8
-                   clang-format-3.7
-                   clang-format-3.6
-                   clang-format-3.5
-                   clang-format-3.4
-                   clang-format-3.3
+             # 3.3-3.9, 4.0-9.0 have been removed because they aren't needed by PCL
+                   clang-format-11
```

Comment:
Does it make sense to be future-proof and add 12, 13, ... already to the list?

Reply:
I'll just add it. Because why not.